integration/utils/worker_stats_tracker_test.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestWorkerStatsTracker creates a new worker stats,
+// adds a couple of transactions there and verifies that average tps is correct.
+func TestWorkerStatsTracker(t *testing.T) {
+	st := NewWorkerStatsTracker()
+	st.StartPrinting(time.Second)
+	defer st.StopPrinting()
+
+	st.AddWorker()
+
+	assert.Equal(t, st.AvgTPSBetween(time.Now().Add(-time.Hour), time.Now()), 0.)
+	st.AddTxSent()
+	st.AddTxSent()
+	assert.Greater(t, st.AvgTPSBetween(time.Now().Add(-time.Hour), time.Now()), 0.)
```

Comment:
i forget how this is calculated, is it easy enough to calculate exactly what this would return? like 2/3600s or is it calculated differently than that?

Reply:
Good point.  I generally try to make any `time.*`-dependent tests to be veeeeery permissive, but here I think I can be slightly more aggressive.